actions/chains/bgp_prefix_exceeded_chain.yaml

```diff
@@ -0,0 +1,75 @@
+---
+  chain:
+      -
+        name: "show_bgp_neighbours"
```

Comment:
@robwwd seconded. Prince Charles himself was one of the original authors of NAPALM and he would be most upset if American English were used.